_ml-commons-plugin/agents-tools/tools/index.md

```diff
@@ -33,6 +33,7 @@ Each tool takes a list of parameters specific to that tool. In the preceding exa
 |:---	|:---	|
 |[`AgentTool`]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/tools/agent-tool/)	|Runs any agent. |
 |[`CatIndexTool`]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/tools/cat-index-tool/)	|Retrieves index information for the OpenSearch cluster. |
+|[`ConnectorTool`]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/tools/connector-tool/)	|Run execute action in connector. |
```

Comment:
"Runs the execute action in a connector"?

Reply:
Reworded.